crates/bevy_utils/macros/src/lib.rs

```diff
@@ -175,3 +226,72 @@ pub fn all_tuples_with_size(input: TokenStream) -> TokenStream {
         )*
     })
 }
+
+/// Parses the attribute `#[doc(fake_variadic)]`
+fn parse_fake_variadic_attr(input: ParseStream) -> Result<bool> {
+    let attribute = match input.call(Attribute::parse_outer)? {
+        attributes if attributes.is_empty() => return Ok(false),
+        attributes if attributes.len() == 1 => attributes[0].clone(),
+        attributes => {
+            return Err(Error::new(
+                input.span(),
+                format!("Expected exactly one attribute, got {}", attributes.len()),
+            ))
+        }
+    };
+
+    if attribute.path().is_ident("doc") {
+        let nested = attribute.parse_args::<Ident>()?;
+        if nested == "fake_variadic" {
+            return Ok(true);
+        }
+    }
+
+    Err(Error::new(
+        attribute.meta.span(),
+        "Unexpected attribute".to_string(),
+    ))
+}
+
+fn to_ident_tuple(idents: impl Iterator<Item = Ident>, len: usize) -> TokenStream2 {
```

Comment:
Nit: order this below the `choose_...` function

Reply:
Good idea :)
I also just noticed that I forgot to replace the original occurrences of the function.